### PR TITLE
Fixed case when identifiers array is empty

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

File: Form/FieldTypeHandler/EnhancedSelection.php [hunk 1]
```diff
@@ -67,7 +71,7 @@ protected function buildFieldForm(
         $fieldSettings = $fieldDefinition->getFieldSettings();
         $optionsValues = $fieldSettings['options'];
 
-        $options['expanded'] = false;
+        $options['expanded'] = true;
```
Comment (emodric) [inline Form/FieldTypeHandler/EnhancedSelection.php]: Why this?

**Comment by emodric** [comment]
Thanks @MarioBlazek!

Anything in this that can be covered by unit test? ;)

File: Form/FieldTypeHandler/EnhancedSelection.php (hunk 1 above)
Comment (MarioBlazek) [inline Form/FieldTypeHandler/EnhancedSelection.php]: Now by default checkboxes will be displayed instead of radio buttons.

Comment (emodric) [inline Form/FieldTypeHandler/EnhancedSelection.php]: I know that, but it's not related to this PR, and moreover, legacy type displays `<select>` element and does not have the option to display checkboxes and radio buttons, so we should keep it consistent.

**Pull request merged**